cliqz.cfg

```diff
@@ -207,6 +207,7 @@ if (getenv('MOZ_CLIQZ_PRIVATE_MODE')) {
   pref("browser.pocket.api", "");
   pref("browser.pocket.site", "");
   pref("network.http.referer.hideOnionSource", true);
+  pref("privacy.resistFingerprinting.block_mozAddonManager", true); // Prevent mozaddonmanager on AMO | DB-1879
```

Comment:
`DB-1879` looks like an internal ticket. It is not very helpful here, since the ticket system is internal only.